doc/source/basic-use.rst

```diff
@@ -1,153 +1,259 @@
+.. index:: Basic Usage
+.. _Basic Usage:
+
+Basic Usage
 ===========
-Basic usage
-===========
 
-This section is about basic use and is a suitable quick howto for newcomers.
+This section is about basic iocage usage and is meant as a "how-to"
+reference for new users.
+
+.. tip:: Remember, command line help is always available by typing
+   :command:`iocage --help` or :command:`iocage [subcommand] --help`.
+
+iocage has a basic "flow" when first used. As a new user interacts with
+iocage for the first time, this flow guides them through initializing
+iocage, then interacting with newly created jails.
+
+.. index:: Activate iocage
+.. _Activate iocage:
 
-Fetch a release
+Activate iocage
 ---------------
 
-The very first step with iocage is to fetch a RELEASE. By default iocage will attempt to fetch the
-host's current RELEASE from the freebsd.org servers. Once the RELEASE bits are downloaded, the most
-recent patches are applied too.
+Before iocage is functional, it needs to :command:`activate`.
+Essentially, iocage needs to link with a usable zpool. In most cases,
+activation is automatic to the primary system zpool, but more advanced
+users can use :command:`iocage activate` to designate a different zpool
+for iocage usage.
+
+Once iocage is ready with an active zpool, users are able to immediately
+begin downloading FreeBSD releases for jail creation.
+
+.. index:: Fetch a release
+.. _Fetch a Release:
+
+Fetch a Release
+---------------
 
-Simply run:
+iocage now needs to fetch a RELEASE, which is used to create jails. By
+default, typing :command:`iocage fetch` opens a menu for the user to
+choose which release to download, as seen in this example:
 
-``iocage fetch``
+.. code-block:: none
 
-If a specific RELEASE is required run:
+ # iocage fetch
+ [0] 9.3-RELEASE (EOL)
+ [1] 10.1-RELEASE (EOL)
+ [2] 10.2-RELEASE (EOL)
+ [3] 10.3-RELEASE
+ [4] 11.0-RELEASE
 
-``iocage fetch release=9.3-RELEASE``
+ Type the number of the desired RELEASE
+ Press [Enter] to fetch the default selection: (default)
```

Comment:
We should still show the default as `11.0-RELEASE`, so the user knows it is capable of autofilling with useful information and not just `default`.